jetty-http2/http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Session.java

```diff
@@ -1250,35 +1269,47 @@ private DataEntry(DataFrame frame, IStream stream, Callback callback)
             // of data frames that cannot be completely written due to
             // the flow control window exhausting, since in that case
             // we would have to count the padding only once.
-            dataRemaining = frame.remaining();
+            dataBytes = frame.remaining();
+        }
+
+        @Override
+        public int getFrameBytesLeft()
```

Comment:
use `Remaining` rather than `Left`

Reply:
Actually is it Remaining?  updateFrameBytes adds updates, so it is how many have been written, not how many remaining to write?